source/reference/code-blocks.txt

```diff
@@ -0,0 +1,467 @@
+=============
+Code Examples
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Snooty supports the following directives for code samples:
+
+- ``code-block``
+- ``io-code-block``
+- ``literalinclude``
+
+These directives feature a monospace font and syntax highlighting that makes
+source code easier to interpret.
+
+Code Blocks
+-----------
+
+The ``code-block`` directive formats a code sample with the following options.
+
+Syntax
+~~~~~~
+
+.. code-block::
+
+   .. code-block:: <language>
+      :copyable: <boolean>
+      :emphasize-lines: <line numbers>
+      :linenos:
+      :caption: <caption text>
+      :source: <url>
+
+      <code sample>
+
+Options
+~~~~~~~
+
+All of the following options are optional.
+
+.. list-table::
+
+   * - ``<language>``
+     - Language to use for syntax highlighting. 
+       
+       For a complete list of supported languages, see the
+       `leafygreen-ui GitHub repository. <https://github.com/mongodb/leafygreen-ui/tree/main/packages/code#properties>`__
```

Comment:
Issue:
I think this link shows the code for rendering the HTML of a code component and not the supported languages.

Suggestion:
I think https://github.com/mongodb/leafygreen-ui/blob/main/packages/code/src/languages.ts lists the supported languages.